definitions/pipelines/gathered_cle_aml_trio.cwl

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env cwl-runner
```

Comment:
just to make things line up nicely when looking through directories/lists of pipelines, it might be good to name this file "cle_aml_trio_gathered.cwl", so that it's right next to the original...

Reply:
Hmm, somehow I have an impression that all gather cwls are named after gathered_xxxxxx.cwl

Reply:
I think they are, but I'm saying that we should change that ;)